pages/linux/aptitude.md

```diff
@@ -23,7 +23,7 @@
 
 `aptitude remove {{package}}`
 
-- Upgrade installed packages to newest available versions:
+- Upgrade installed packages to the newest available versions:
```

Comment:
Possibly, `their` might be better.
```suggestion
- Upgrade installed packages to their newest available versions:
```

Reply:
I'd say `the` is better. Why give it personification?